examples/src/test/kotlin/CodecTest.kt

```diff
@@ -0,0 +1,212 @@
+
+import com.mongodb.MongoClientSettings
+import com.mongodb.kotlin.client.coroutine.MongoClient
+import io.github.cdimascio.dotenv.dotenv
+import kotlinx.coroutines.flow.toList
+import kotlinx.coroutines.runBlocking
+import org.bson.BsonReader
+import org.bson.BsonType
+import org.bson.BsonWriter
+import org.bson.codecs.*
+import org.bson.codecs.configuration.CodecProvider
+import org.bson.codecs.configuration.CodecRegistries
+import org.bson.codecs.configuration.CodecRegistry
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import kotlin.test.assertEquals
+
+
+// :replace-start: {
+//    "terms": {
+//       "CONNECTION_URI_PLACEHOLDER": "\"<connection string uri>\""
+//    }
+// }
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+internal class CodecTest {
```

Comment:
Is this intended to be reviewed?  It's not referenced in the FAQ.

Reply:
ohh, this is a git error on my part. i'll delete that from the PR. thx for spotting.

Reply:
fixed